src/components/CaseEditor/CaseEditor.scss

```diff
@@ -52,8 +89,13 @@
       text-align: center;
       &.lead {
         border: 4px solid $interactive-color;
+        .trash {
+          bottom: 1px;
+          border: none;
+        }
         .checkbox {
-          background-image: url("../../img/check.png");
+          border: none;
+          background-image: url("../../img/star.png");
```

Comment:
I think we're better off using SVG icons so we have scalability, etc..  See https://github.com/participedia/frontend/blob/448ae34b338bb43dfc16a33ad12b7c914bca7963/src/Upload.js#L78 for an example.